src/views/SignIn/SignIn.js

```diff
@@ -137,6 +137,8 @@ const SignIn = ({ history, loginUserAction, setCurrentUserAction }) => {
 
   useEffect(() => {
     const errors = validate(formState.values, schema);
+    var token = localStorage.getItem('jwtToken', token);
```

Comment:
1. var' used instead of 'let' or 'const'  -> **Use const.**
2.  Invalid number of arguments, expected 1-> The second argument token is unnecessary

Reply:
Done